spec/api-browser-window-spec.ts

```diff
@@ -3586,6 +3595,7 @@ describe('BrowserWindow module', () => {
 
       after(() => {
         server.close();
+        server = null as unknown as http.Server;
```

Comment:
We shouldn't need all these dereferences, it doesn't change if sockets are retained. Can leave them in with reason but otherwise cleaning them back up would be good IMO